License.md

```diff
@@ -6,10 +6,10 @@ For more information and requests about licensing please [contact us through our
 
```

Comment:
Please remove the line 5 above. ([see](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/166/files))